dll/win32/comctl32/imagelist.c

```diff
@@ -841,6 +845,7 @@ ImageList_Create (INT cx, INT cy, UINT flags,
     himl->clrFg     = CLR_DEFAULT;
     himl->clrBk     = CLR_NONE;
     himl->color_table_set = FALSE;
+    himl->usVersion = 0;
```

Comment:
```suggestion
#ifdef __REACTOS__
    himl->usVersion = 0;
#endif
```
Does this need the ifdef there too?